docs/editor_guide/creating_personalized_content.rst

```diff
@@ -0,0 +1,91 @@
+Creating personalized content
+=============================
+
+Once you've created a segment you can start serving personalised content to your
+visitors. To do this, you can go one of three directions.
+
+1. Create a variant of a page for your segment.
+2. Use StreamField blocks only visible to your segment.
+3. Use a template block only visible to your segment.
+
+
+Method 1: Create a variant
+--------------------------
+
+**Why you'd want to use this method**
+
+* It has absolutely no restrictions, change anything you want.
+* That's pretty much it.
+
+**Why you'd want to use a different method**
+
+* You are editing a page that changes often. You'd probably rather not change
+  variant every time the original page changes.
+
+To create a variant from a page for a specific Segment (which you can change to
+your liking after creating it), simply go to the Explorer section and find the
+page you'd like to personalize.
+
+.. figure:: ../_static/images/variants_button.png
+   :alt: The variants button that appears on personalisable pages.
+
+When you hover over a page, you'll notice a "Variants" dropdown button appears.
+Click the button and select the segment you'd like to create personalized
+content for.
+
+Once you've selected the segment, a copy of the original page will be created
+with a title that includes the segment. Don't worry, your visitors won't be able
+to see this title. It's only there for your reference.
+
+.. figure:: ../_static/images/editing_variant.png
+   :alt: The newly created page allowing you to change anything you want.
+
+You can change everything on this page you'd like. Visitors that are appointed
+to your segment will automatically see the new variant you've created for them
+when attempting to visit the original page.
+
+
+Method 2: Use a StreamField block
+---------------------------------
+
+Preparing a page and it's StreamField blocks for this method is described in the
+Usage guide for developers. Please refer to
+:ref:`implementing_streamfield_blocks` for more information.
+
+**Why you'd want to use this method**
+
+* Allows you to create personalised content in the original page (without
+  creating a variant).
+* Create multiple StreamField blocks for different segments inline.
+
+**Why you'd want to use a different method**
+
+* You need someone tech savvy to change the back-end implementation.
+
+To create personalised stream field blocks, first select the page you wan't to
```

Comment:
misschien ff een find/replace op ``personalis`` of ``personaliz``, staan er verderop namelijk nog meer